### ISSUE TITLE
Tabbing onto a graphic needs to be apparent to screen reader users

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-browser#28]
The tabbing to images even if they wouldn't normally be a tab stop appears to work well (implemented in #22). However, there are numerous images that when I tab to them, I hear nothing in NVDA. I think I have some limitations in my understanding of how people navigate these pages (wikipedia, cnn, my MIMIC webpage at CIM, etc), since it isn't just images where I hear nothing, but also the links in the wikipedia article I looked at, for example.

@Sabrina-Knappe I was planning to assign this directly to @JRegimbal to implement something that would cause screen readers to just say something like "image" for every image we put a tabstop onto, but I want to make sure I'm not missing something about how these pages are actually navigated, to make sure this is a reasonable thing to do?

**Comment by Sabrina-Knappe** [comment]
Has this been successfully dealt with? I tested with both NVDA and VoiceOver, and images (for the most part) are being stopped at. I found a really good example site for this but I think it was on my other computer. I will send you the link once I have it.

**Comment by Sabrina-Knappe** [comment]
Here is the relevant Alt text image page:
https://webaim.org/articles/nvda/images.htm
